strr-examiner-web/app/components/ApplicationInfoHeader.vue

```diff
@@ -109,6 +113,14 @@ const registrationCountdown = computed(() => {
       </div>
       <div class="mb-2 text-sm">
         <UBadge
+          v-if="activeHeader.isSetAside"
+          class="mr-3 bg-[#495057] font-bold uppercase"
```

Comment:
Shouldn't we be using bcGovColor-midGray instead of #495057 directly?
https://github.com/bcgov/sbc-connect-common/blob/main/nuxt/layers/core/tailwind.config.ts

Reply:
Yea, agree, I will update it.